src/main/java/com/artipie/api/artifactory/AddUpdatePermissionSlice.java

```diff
@@ -142,6 +143,7 @@ public Response response(final String line, final Iterable<Map.Entry<String, Str
                     )
                 )
         );
+        res.add(new RepoPermissions.PermissionItem("/readers", new ListOf<String>("read")));
```

Comment:
@olenagerasimova `/readers` group should be added only if new repository is created. Otherwise it won't be possible to revoke permissions for the group

Reply:
@olegmoz are you sure such behavior will be correct from our colleagues point of view? As far as I understood, users always have `read` permission

Reply:
@olenagerasimova I tried to lookup requirements about it and cannot really find direct answer. So let's assume `readers` group is always present in repository and all users are in `readers` group.

Still there is a problem in this line: we need to add `readers` group to the list only if it is absent in request. Otherwise it is going to be duplicated if user is loaded and the saved as-is. The same goes for adding `readers` group to user in `AddUpdateUserSlice` class.